backend/instancer/api/account.py

```diff
@@ -336,6 +336,30 @@ def login() -> ResponseReturnValue:
     }
 
 
+@blueprint.route("/logout", methods=["POST"])
+def logout() -> ResponseReturnValue:
+    """Logout the current user
+
+    Requires body param login_token"""
+    try:
+        token = request.form["login_token"]
```

Comment:
Probably best to instead make the form field to just `token` or `account_token` or `session_token`, or better yet, read the token from the authentication header directly since that field is actually checked for validity before this function is ran (though do not assume the token is still valid due to possible race conditions)

Reply:
For simplicity and mimic the style of the other API endpoints, I'll go with just `token`.